Riot/Modules/Room/NotificationSettings/SwiftUI/FormPickerItemView.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct FormPickerItemView: View {
```

Comment:
Yea, I think you are right its a prime candidate. Common UI component libraries can become a bit of a dumping ground in some cases which can really affect their usefulness. I think it might be good to have a light process for committing to shared components and some ways of keeping them in sync with design team.